src/components/Footer.css

```diff
@@ -135,8 +135,9 @@
   .footer-logo {
     z-index: 1;
     display: flex;
-    width: 130px; /* Not the cleanest fix sorry but it works as long as we don't resize the logo*/
+    width: 150px; /* Not the cleanest fix sorry but it works as long as we don't resize the logo*/
     height: 50px;
+    margin-right: 180px;
```

Comment:
This may cause issues in mobile view/different screen resolutions if not handled under line 149